setup.py

```diff
@@ -261,55 +236,75 @@ def run(self):
 
 class BuildPy(setuptools.command.build_py.build_py):
     def run(self):
-        self.run_command("create_version")
-        self.run_command("cmake_build")
-
-        generated_python_files = glob.glob(
-            os.path.join(CMAKE_BUILD_DIR, "onnx", "*.py")
-        ) + glob.glob(os.path.join(CMAKE_BUILD_DIR, "onnx", "*.pyi"))
-
-        for src in generated_python_files:
-            dst = os.path.join(TOP_DIR, os.path.relpath(src, CMAKE_BUILD_DIR))
-            self.copy_file(src, dst)
-        # Review the command customizations to enable editable_mode
-        self.editable_mode = False
-        return setuptools.command.build_py.build_py.run(self)
+        self.create_version()
+        return super().run()
+
+    def create_version(self):
+        if self.editable_mode:
```

Comment:
```suggestion
        # We do not make create_version into its own command because we need to use self.build_lib
        if self.editable_mode:
```